src/sankey/index.js

```diff
@@ -74,39 +77,40 @@ class Sankey extends Component {
     const nWidth = sankeyInstance.nodeWidth();
     const path = sankeyInstance.link();
 
-    const voronoiInstance = voronoi()
-      .x(d => d.x + d.dx / 2)
-      .y(d => d.y + d.dy / 2)
-      .extent([[-margin, -margin], [width + margin, height + margin]]);
+    // Create a voronoi with each node center points
+    const voronoiInstance = hasVoronoi ?
+      voronoi()
+        .x(d => d.x + d.dx / 2)
+        .y(d => d.y + d.dy / 2)
+        .extent([[-margin, -margin], [width + margin, height + margin]]) :
+      null;
 
     return (
-      <svg height={height + margin} width={width + margin} className="rv-sankey">
+      <svg height={height + margin} width={width + margin} className={`rv-sankey ${className}`}>
         <g transform={`translate(${margin / 2}, ${margin / 2})`}>
 
           {links.map((link, i) => (
             <path
               d={path(link)}
               className="rv-sankey__link"
-              opacity={link.opacity || defaultLinkOpacity}
-              stroke={link.color || defaultLinkColor}
+              opacity={link.opacity || DEFAULT_LINK_OPACITY}
```

Comment:
make sure you cover opacity in the isNaN? also can you use Number.isFinite? isNaN has some problems